{"uuid":"bc8b14f2-b15e-4517-8f99-c3a8452989cf","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["cc9cac39-5ab9-4436-af8d-26baaf255278"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887587832,"stop":1696887587833},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887587909,"stop":1696887587910},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887587985,"stop":1696887587986},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696887590029,"stop":1696887590030}],"start":1696887587540,"stop":1696887590031}